selfdrive/loggerd/logger.cc

```diff
@@ -94,23 +94,22 @@ kj::Array<capnp::word> logger_build_init_data() {
   init.setDirty(!getenv("CLEAN"));
 
   // log params
-  Params params = Params();
-  init.setGitCommit(params.get("GitCommit"));
-  init.setGitBranch(params.get("GitBranch"));
-  init.setGitRemote(params.get("GitRemote"));
-  init.setPassive(params.getBool("Passive"));
-  init.setDongleId(params.get("DongleId"));
-  {
-    std::map<std::string, std::string> params_map;
-    params.readAll(&params_map);
-    auto lparams = init.initParams().initEntries(params_map.size());
-    int i = 0;
-    for (auto& kv : params_map) {
-      auto lentry = lparams[i];
-      lentry.setKey(kv.first);
-      lentry.setValue(capnp::Data::Reader((const kj::byte*)kv.second.data(), kv.second.size()));
-      i++;
-    }
+  std::map<std::string, std::string> params_map;
+  Params().readAll(&params_map);
+
+  init.setGitCommit(params_map["GitCommit"]);
+  init.setGitBranch(params_map["GitBranch"]);
+  init.setGitRemote(params_map["GitRemote"]);
+  init.setPassive(params_map["Passive"] == "1");
```

Comment:
```suggestion
  init.setPassive(params.getBool("Passive"));
```

Let's keep using the helper function here in case we ever change the definition of what values classify as true.